sdk/textanalytics/azure-ai-textanalytics/src/main/java/com/azure/ai/textanalytics/models/PiiEntityDomainType.java

```diff
@@ -0,0 +1,30 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.ai.textanalytics.models;
+
+import com.azure.core.annotation.Immutable;
+import com.azure.core.util.ExpandableStringEnum;
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+/**
+ * Gets the PII entity domain type. It is provided by service v3.1-preview.2 and later.
+ */
+@Immutable
+public final class PiiEntityDomainType extends ExpandableStringEnum<PiiEntityDomainType> {
```

Comment:
Do we require this to be an expandable enum?

Reply:
oh I missed this. +1 on ^ question

Reply:
Seems not. I will change to regular enum.